docs/apidocs/qiskit_nature.utils.rst

```diff
@@ -0,0 +1,18 @@
+﻿
```

Comment:
Sure. This file was automatically generated by Sphinx so I wasn't sure if I should edit it.

Reply:
Done.

Reply:
When adding a folder that needs to be documented, I have done this enough that I just create one of these, that way Sphinx uses it and will not auto-create a default one. Editing it is fine though - its just something sphinx needs and it did it itself since it wasn't provided it! Thx.